.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: 🔬 Test
+
+on:
+  workflow_dispatch:
```

Comment:
Not sure this will work as intended. Required workflows only magically appear on `pull_request` and other wise _do not exist._